main.go

```diff
@@ -50,24 +49,28 @@ func main() {
 	var metricsAddr string
 	var enableLeaderElection bool
 	var syncPeriod time.Duration
+	var namespace string
+	var controllerNamespace string
+
+	flag.BoolVar(&enableLeaderElection, "enable-leader-election", false, "Enable leader election for controller manager, this will ensure there is only one active controller manager.")
+	flag.DurationVar(&syncPeriod, "informer-re-sync-interval", 10*time.Second, "controller shared informer lister full re-sync period")
 	flag.StringVar(&metricsAddr, "metrics-addr", ":38080", "The address the metric endpoint binds to.")
-	flag.BoolVar(&enableLeaderElection, "enable-leader-election", false,
-		"Enable leader election for controller manager. "+
-			"Enabling this will ensure there is only one active controller manager.")
-	flag.DurationVar(&syncPeriod, "informer-re-sync-interval", 10*time.Second,
-		"controller shared informer lister full re-sync period")
+	flag.StringVar(&namespace, "namespace", "", "Namespace to watch for resources, defaults to all namespaces")
+	flag.StringVar(&controllerNamespace, "controller-namespace", "vela-system", "Namespace to run the terraform jobs")
```

Comment:
```suggestion
	flag.StringVar(&controllerNamespace, "controller-namespace", "terraform", "Namespace to run the terraform jobs")
```
Let make it to be consistent with https://github.com/oam-dev/terraform-controller/blob/master/getting-started.md#prerequisites.